### ISSUE TITLE
[SECURITY] Vulnerabilities in .dll and .exe files

### ISSUE THREAD

**Issue by JeromeGaudin** [leokhoa/laragon#315]
There is many vulnerabilities in .dll and .exe files. Could you please update each vulnerables libraries to the last version.
![image](https://user-images.githubusercontent.com/31897154/176174005-b11a7c52-f562-432f-b32b-59bffb550ec3.png)
You can have the same result with the free tool OWASP Dependency Check https://owasp.org/www-project-dependency-check/.

**Comment by stgoos** [comment]
@JeromeGaudin can we patch these vulnerabilities ourselves in the meantime?

But fully agree with you, this should be patched asap in the downloadable versions of Laragon.

**Comment by DigiLive** [comment]
Many of them come from user configurable packages like php.
Although I agree some vulnerabilities should be removed by an update of Laragon, the other vulnerabilities could/should be removed by the user itself by installing appropriate versions of the currently installed user packages.
E.g. php, apache, etc.

**Comment by stgoos** [comment]
Agreed. I ended up updating everything within my power in July and made it a regular thing to do so every 2 months (added a reminder in my calendar for it).

That said, an update of the Laragon core would be very useful to keep more people safe(r).

**Comment by alxndr-w** [comment]
I was not sure if I should close this issue as Laragon 7 was released. And @stgoos pointed out to self-update librarys is always an option.

Even there can be critical bugs in e.g. PHP, first of all, they aren't critical as long as you don't give strangers or untrustworthy code a chance to execute. Then, they don't need Laragon or critical PHP bugs to damage your PC. 

As far as I know, Laragon is not meant to be used in production. I wouldn't recommend XAMPP or MAMP either for that task.
